src/walk.js

```diff
@@ -0,0 +1,12 @@
+import * as walk from 'acorn/dist/walk';
+import { DynamicImportKey } from './inject';
+
+export function inject(injectableWalk) {
+  return Object.assign({}, injectableWalk, {
+    base: Object.assign({}, injectableWalk.base, {
+      [DynamicImportKey]: () => {},
```

Comment:
Rather than using an arrow function here you could also just use object function shorthand:
```js
{
  [DynamicImportKey]() {},
}
```